processor/error/event.go

```diff
@@ -165,13 +166,15 @@ func (e *Event) addException(config config.Config, service m.Service) {
 	utility.Add(ex, "type", e.Exception.Type)
 	utility.Add(ex, "handled", e.Exception.Handled)
 
-	switch e.Exception.Code.(type) {
+	switch code := e.Exception.Code.(type) {
 	case int:
```

Comment:
I don't think this case will ever be hit, will it? Without UseNumber, it'll be string or float64; with UseNumber, it'll be string or json.Number.

Reply:
I think you're right, this probably can be removed.